common/persistence/nosql/nosqlplugin/cassandra/tasks.go

```diff
@@ -373,7 +373,7 @@ func (db *cdb) InsertTasks(
 				task.WorkflowID,
 				task.RunID,
 				scheduleID,
-				tasklistCondition.LastUpdatedTime,
+				task.CreatedTime,
```

Comment:
same as above.

Reply:
This is different from the above one and needs to be fixed & released. 

It should use the same value as the no ttl case above.

Reply:
yeah, I meant we need to add an assertion in the persistence test so that it wont be broken again.

Reply:
so this is a bit complex. there's nothing we need to add in the test since there's already a check for task create time. The problem is that in tests, when calling the CreateTask API, we provided a tasklist info with LastUpdateTime, but in application code when calling CreateTask, we don't have the LastUpdateTime so the value is empty. That's why previously the test is also passing. 

For now, I changed the test and cleared the tasklist LastUpdatedTime field when calling CreateTask in persistence tests.

Reply:
I see. So the test was cheating..